packages/slo-stat-bot/src/slo-bot.ts

```diff
@@ -40,17 +63,16 @@ async function handleIssues(
   owner: string,
   repo: string,
   type: string,
+  number: number,
```

Comment:
This is probably too many parameters for a function to be easily readable.  It feels like many of these fields would be available on an `Issue` interface.  I'm also guessing in the `Octokit` namespace there would be a `Issue` type we could readily use here.

Reply:
I agree; I suggested she do a second PR to do that fix in order to keep this PR relatively small and stable. Let us know if you want to add this to this PR though

Reply:
As Colin mentioned I just created the new PR which includes the issue types: https://github.com/googleapis/repo-automation-bots/pull/795